parsers/handleWordpressComponents.js

```diff
@@ -74,6 +74,56 @@ export function fixAccordions() {
   };
 }
 
+const findCaptionText = (figureCaption, key) => {
+  let figCaptionValue = '';
+  const nestedFigCap = figureCaption.filter((o) => o.type === key);
+  if (nestedFigCap.length > 0) {
+    figCaptionValue = nestedFigCap[0].value;
+  } else {
+    const string = findCaptionText(figureCaption[0].children, key);
+    if (string) {
+      figCaptionValue += string;
+    }
+  }
+  return figCaptionValue;
+};
+
+export function findImageData() {
+  return (tree) => {
+    visit(tree, (node) => node.type === 'element' && node.children.some((n) => n.tagName === 'img'), (node) => {
+      const arrayOfImageData = node.children.filter((imageData) => imageData.tagName === 'img');
+      const { src } = arrayOfImageData[0].properties;
+      const { alt } = arrayOfImageData[0].properties;
```

Comment:
You can do
```
const { src, alt } = arrayOfImageData[0].properties;
```
Out of curiosity, that array will never have more than one element, right? Might be worth running it for all our in files just to be sure.

Reply:
That's right yeah but i tried running all the files before hand to make sure it was doing as expected for TF, edt etc.